### DISCUSSION TITLE
Discord Game Activity

### DISCUSSION THREAD

**Discussion by sillitatti** [netblue30/firejail#4131] [Q&A] [answered]
How might Discord's Game Activity detection be made to work with firejail? Even if I run Discord with "firejail --noprofile Discord", this what I get:

![kuva](https://user-images.githubusercontent.com/49153348/111837442-8d62cd80-8900-11eb-951e-3692198cff89.png)

I'm running Firejail 0.9.64 on Fedora 33. Discord build number 80041.

**Comment by rusty-snake** [comment]
> Even if I run Discord with "firejail --noprofile Discord", this what I get:

Try if the [noprofile.profile](https://gist.github.com/rusty-snake/bb234cb3e50e1e4e7429f29a7931cc72) helps. If not there is no hope.

> Firejail 0.9.64 

is vulnerable to CVE-2021-26910.

**Comment by sillitatti** [comment]
> Try if the [noprofile.profile](https://gist.github.com/rusty-snake/bb234cb3e50e1e4e7429f29a7931cc72) helps. If not there is no hope.

Thanks, I tried it. Same result 😞

**Comment by reinerh** [comment, marked as answer]
Some time ago I worked on an AppArmor profile for Discord and noticed in the audit log that it tries to access the process list, and even to ptrace some processes (browsers, steam, spotify, ...).

@rusty-snake mentioned in the noprofile that a new PID namespace is still being created. So Discord will not see other processes.

What you could try is to start the other processes with firejail as well and join their namespaces (`--join=$pid`).

**Comment by sillitatti** [comment]
> What you could try is to start the other processes with firejail as well and join their namespaces (`--join=$pid`).

Thank you! It works with the examples `firejail --noprofile --name=steam steam` and `firejail --join=steam Discord`. Now I'll just need to create a good combined profile.
